usaon_benefit_tool/templates/home.md

```diff
@@ -0,0 +1,43 @@
+{# Thinking about how this should be due to the way that it links to Benefit Tool docs #} 
+
+## {% block title %}Welcome to the US AON Benefit Tool!{% endblock %}
```

Comment:
This is Jinja weirdness to indicate that this is the "title" of the current page. We could pull this out in to the parent HTML template instead? :shrug: